source/manage-apps/configure/config/triggers.txt

```diff
@@ -202,7 +203,16 @@ configuration files:
           preimages for the collection.
           
           For more information, see :ref:`preimages`.
-       
+
+   * - | ``config.tolerate_resume_errors``
+       | Boolean
+     - If ``true``, the Trigger automatically resumes after an error.
```

Comment:
I am not sure if this line is 100% clear. The trigger will not automatically resume after all errors, only specifically resume token errors. The section below (included from `trigger-auto-resume.rst`) clarifies this, but I would be a little nervous that people might hope that this auto resumes on any error.

Reply:
thanks good catch. will rephrase accordingly.